bin/NpmReleaseCommand.re

```diff
@@ -266,16 +275,16 @@ let makeBinWrapper = (~destPrefix, ~bin, ~environment: Environment.Bindings.t) =
       let findVarRe = Str.regexp "\\$\\([a-zA-Z0-9_]+\\)" in
       let replace v =
         let name = Str.matched_group 1 v in
-        try Hashtbl.find curEnvMap name
-        with Not_found -> ""
+        let values = EnvHashtbl.find_all curEnvMap name in
+        String.concat ";" values
```

Comment:
I don't think this change is needed — esy generates `PATH=$PATH` as its first binding for `PATH` and thus initial `PATH` is included in the env. Did you find otherwise? What's the purpose of this change?

Reply:
My thought was that there might be `Path`, `path` and `PATH` but that might not be an issue?

I'll try to revert to previous and update if it works.

Reply:
Works on Windows, pushed the change.

Thanks a lot for reviewing this 😄